www/main.js

```diff
@@ -1,46 +1,48 @@
 ﻿$(function () {
     var roomArray = window.location.pathname.split('/');
-    roomName = roomArray[2];
+    var roomName = roomArray[2];
```

Comment:
Probably end up with some weird side effects from roomName being global. Better to pass it around when needed.